src/extension.ts

```diff
@@ -9,14 +9,19 @@ import * as VSCode from "vscode";
 import Program from "./Program";
 import ResourceManager from "./Properties/ResourceManager";
 import Settings from "./Properties/Settings";
+import { getMarkdownExtensionContributions } from "./System/Drawing/MarkdownExtensions";
 import Exception from "./System/Exception";
 import MarkdownFileNotFoundException from "./System/MarkdownFileNotFoundException";
 
+var markdown;

```

Comment:
It's because `this` aka. `outside` is type of `any` which can have whatever properties it want.  
Also the `var markdown;` code-line doesn't belong to the `this`-context anyway... whyever... I don't get ECMAScript xD

You could, btw, do something like this here:

```ts
let outside: { markdown: markdownIt.MarkdownIt; } = this;
```